From 6192e72a570ef0120691ef215115e3316aa97a43 Mon Sep 17 00:00:00 2001
From: Nikolay Aleksandrov <nikolay@redhat.com>
Date: Mon, 1 Sep 2014 13:33:11 -0400
Subject: [crypto] qat: checkpatch blank lines

Message-id: <1409578401-26943-20-git-send-email-nikolay@redhat.com>
Patchwork-id: 90096
O-Subject: [RHEL7.1 PATCH BZ 1091130 19/29] crypto: qat - checkpatch blank lines
Bugzilla: 1091130
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Herbert Xu <herbert.xu@redhat.com>

From: Tadeusz Struk <tadeusz.struk@intel.com>

Fix new checkpatch hits:
CHECK:LINE_SPACING: Please use a blank line after
function/struct/union/enum declarations

Signed-off-by: Bruce Allan <bruce.w.allan@intel.com>
Signed-off-by: Tadeusz Struk <tadeusz.struk@intel.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
(cherry picked from commit 53275baa0368945ba4f385f2e41918bedd847dce)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/crypto/qat/qat_common/adf_transport_internal.h b/drivers/crypto/qat/qat_common/adf_transport_internal.h
index 4eb8969..f854bac 100644
--- a/drivers/crypto/qat/qat_common/adf_transport_internal.h
+++ b/drivers/crypto/qat/qat_common/adf_transport_internal.h
@@ -104,12 +104,15 @@ static inline int adf_bank_debugfs_add(struct adf_etr_bank_data *bank)
 {
  return 0;
 }
+
 #define adf_bank_debugfs_rm(bank) do {} while (0)
+
 static inline int adf_ring_debugfs_add(struct adf_etr_ring_data *ring,
            const char *name)
 {
  return 0;
 }
+
 #define adf_ring_debugfs_rm(ring) do {} while (0)
 #endif
 #endif
diff --git a/drivers/crypto/qat/qat_common/icp_qat_fw_la.h b/drivers/crypto/qat/qat_common/icp_qat_fw_la.h
index d1d4802..c8d2669 100644
--- a/drivers/crypto/qat/qat_common/icp_qat_fw_la.h
+++ b/drivers/crypto/qat/qat_common/icp_qat_fw_la.h
@@ -358,6 +358,7 @@ struct icp_qat_fw_la_resp {
  uint64_t opaque_data;
  uint32_t resrvd[ICP_QAT_FW_NUM_LONGWORDS_4];
 };
+
 #define ICP_QAT_FW_CIPHER_NEXT_ID_GET(cd_ctrl_hdr_t) \
  ((((cd_ctrl_hdr_t)->next_curr_id_cipher) & \
    ICP_QAT_FW_COMN_NEXT_ID_MASK) >> (ICP_QAT_FW_COMN_NEXT_ID_BITPOS))
-- 
1.7.1